### PR TITLE
Update compass cal page to new compass function and MP screens

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: common/source/docs/common-compass-calibration-in-mission-planner.rst [hunk 1]
```diff
@@ -22,30 +22,25 @@ Calibration first steps
...
-   You may wish to disable any internal compasses if you are consistently seeing the "inconsistent compasses" pre-arm message often and you are sure that the external compass is calibrated and the orientation is correct.
+   You may wish to disable any internal compasses if you are consistently seeing the "inconsistent compasses" pre-arm message often and you are sure that the external compass is calibrated. 
 
 .. _onboard_calibration:
 
 Onboard Calibration
 ===================
 
-Copter-3.4, Plane 3.4.0, and Rover 3.0.0 (and higher) support "Onboard Calibration" meaning that the calibration routine runs on the autopilot.  This method is more accurate than the older "Offboard Calibration" (aka "Live Calibration") which runs on the ground station because in addition to offsets, scaling is also calculated.
+"Onboard Calibration" is a calibration routine that runs on the autopilot.  This method is more accurate than the older "Offboard Calibration" (aka "Live Calibration") which runs on the ground station because in addition to offsets, scaling and orientation is also automatically determined.
```
Comment (rmackay9) [inline common/source/docs/common-compass-calibration-in-mission-planner.rst:39]: I think it should be "**are** also automatically determined".

File: common/source/docs/common-compass-calibration-in-mission-planner.rst [hunk 2]
```diff
@@ -87,12 +81,22 @@ If calibration fails:
...
+Compass Ordering
+================
+
+At the top of the page, you can change the priority of the attached compasses, if desired.
+
 Offboard Calibration (aka "Live Calibration")
 =============================================
 
-.. note:: This method is not recommended if you are using a recent version of ArudPilot that supports the Onboard Calibration described above.
+.. note:: This method is not recommended if you are using a recent version of ArudPilot that supports the Onboard Calibration described above. It is no longer available in Mission Planner.
```
Comment (rmackay9) [inline common/source/docs/common-compass-calibration-in-mission-planner.rst:97]: Maybe we should just zap this whole part if it's gone from MP?  .. and remove any other references to "Live" calibration.  happy days!

Comment (Hwurzburg) [reply common/source/docs/common-compass-calibration-in-mission-planner.rst:97]: Fine by me!!gone!

File: common/source/docs/common-compass-calibration-in-mission-planner.rst (hunk 1 above)
Comment (Hwurzburg) [reply common/source/docs/common-compass-calibration-in-mission-planner.rst:39]: yep

File: common/source/docs/common-compass-calibration-in-mission-planner.rst [hunk 3]
```diff
@@ -103,7 +107,7 @@ Older versions of ArduPilot rely on the ground station to calculate the compass
 
       Mission Planner: Live Compass Calibration
 
-- The aim is to rotate the vehicle so that the coloured trail hits each of the white dots.  One way to do this is to hold the vehicle in the air and rotate it slowly so that each side (front, back, left, right, top and bottom) points down towards the earth for a few seconds in turn.
+- The aim is to rotate the vehicle so that the colored trail hits each of the white dots.  One way to do this is to hold the vehicle in the air and rotate it slowly so that each side (front, back, left, right, top and bottom) points down towards the earth for a few seconds in turn.
```
Comment (rmackay9) [inline common/source/docs/common-compass-calibration-in-mission-planner.rst:110]: a battle of american english vs british english! :-)  being Canadian I'd leave it with the "u" in there but it's correct either way and you're doing the majority of the typing these days.

Comment (Hwurzburg) [reply common/source/docs/common-compass-calibration-in-mission-planner.rst:110]: my dictionary lol

**Comment by Hwurzburg** [comment]
thanks, merging

**Pull request merged**